src/applications/vaos/components/BackLink.jsx

```diff
@@ -21,67 +21,35 @@ export default function BackLink({
   const location = useLocation();
   const history = useHistory();
   const dispatch = useDispatch();
+  const { cancelAppointmentStatus } = useSelector(getCancelInfo);
 
-  const handleClickGATracker = () => {
-    let status;
-    if (isPendingAppointment) {
-      status = 'pending';
-    } else if (isUpcomingAppointment) {
-      status = 'upcoming';
-    } else if (isPastAppointment) {
-      status = 'past';
-    }
-
-    const progress =
-      location.search === '?confirmMsg=true' ? 'confirmation' : 'appointment';
-
-    if (progress === 'confirmation' && status === 'upcoming') {
-      status = 'direct';
+  let status;
+  let link = '/';
+  let linkText = 'Back to appointments';
+  if (isPendingAppointment) {
+    status = 'pending';
+    link = '/pending';
+    linkText = 'Back to pending appointments';
+    if (
+      featureAppointmentDetailsRedesign &&
```

Comment:
Let's wait until https://github.com/department-of-veterans-affairs/vets-website/pull/31646#pullrequestreview-2280648840 is merged since this will probably be a conflict and we'd like that large PR to be merged first.